packages/catalog-model/src/schema/shared/common.schema.json

```diff
@@ -43,6 +43,11 @@
         },
         "target": {
           "$ref": "#reference"
+        },
+        "targetRef": {
+          "type": "string",
+          "minLength": 1,
+          "description": "The entity ref of the target of this relation."        
```

Comment:
extra blank spaces at the end here :P the linter may complain

Reply:
Aaaaand fixed :)